paper.tex

```diff
@@ -173,13 +174,39 @@ \subsection{Choice of Data Storage}
 
 Using MAM as a data storage benefits from the scalability of the underlying IOTA network as well as the decentralized and fault-tolerant characteristics of distributed ledgers, which reduce the risks of centralized storage services. Furthermore, the rights of data access are traded instead of a copy of data in the trading platform, which eliminates the need for data consumers to have additional storage.
 
+\subsection{Digital Identity}
+The digital identity represents an entity and holds the digital assets within the digital world, it is important to prove and show the identity to others during interactions. We select the decentralized identity model rather than a centralized one for the concerns in Table.\ref{tab:did}.
 
+\begin{table}[h]
+	\caption{The comparism of centralized and decentralized identity model}
+	\label{tab:did}
+	\begin{tabularx}{\linewidth}{|l|X|X|}
+	\hline
+		\textbf{Identity Model} & \textbf{Centralized} & \textbf{Decentralized} \\
+		\hline
+		Control & Enterprises control identities & Entities control their identities \\
+		\hline
+		Security & Identity holded in a centralized service is a honeypot for cyber attacks & Decentralized identity limits data exposure \\
```

Comment:
Use of the word "holded" is a display of incorrect English grammar and sentence formation. The correct word is “held”.

Reply:
oops! :stuck_out_tongue: